src/app/(dashboard)/perfil/page.tsx

```
@@ -1,14 +1,85 @@
+'use client';
+
+import { useQuery } from 'convex/react';
+import { api } from '../../../../convex/_generated/api';
 import { PieChartDemo } from './charts/pie-chart-demo';
 import { ThemeBarChart } from './charts/theme-bar-chart';
+import { StatCard } from './components/stat-card';
+import { Skeleton } from '@/components/ui/skeleton';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)